croncheck.php

```diff
@@ -280,6 +280,10 @@
 // If the Check API from 3.9 exists then call those as well.
 if (class_exists('\core\check\manager')) {
 
+    if (isset($CFG->mnet_dispatcher_mode) and $CFG->mnet_dispatcher_mode !== 'off') {
```

Comment:
if this is a core bug workaround can you add a comment here to the tracker when it gets made

Reply:
I've created https://tracker.moodle.org/browse/MDL-77247
This fails in 4.0 and older only as 4.1 and master have changed.
Added a comment for this.
Cheers

Reply:
Any chance to have a look and have this merged @brendanheywood ?